components/bl-emoji/README.md

```diff
@@ -0,0 +1,47 @@
+</p>
+
+## Properties
+
+| Property                                      | Type                                                               | Default Value   | Logic                        | Data Binding | UI Setting | Description                                                            |
+|-----------------------------------------------|--------------------------------------------------------------------|-----------------|------------------------------|--------------|------------|------------------------------------------------------------------------|
+| Disabled<br/>`disabled`                       | Checkbox                                                           | `false`         | Disabled Logic               | YES          | YES        | This handler allows you to disable a component.                        |
+| Button Color<br/>`buttonColor`                | Color                                                              | `#000000`       | Button Color Logic           | YES          | YES        | This handler allows you to specify the button color.                   |
+| Button Size<br/>`buttonSize`                  | Text                                                               | "24px"          | Button Size Logic            | YES          | YES        | This handler allows you to specify the button size.                    |
+| Dropdown List Height<br/>`dropdownHeight`     | Text                                                               | "500px"         | Dropdown List Height Logic   | YES          | YES        | This handler allows you to specify the dropdown list height.           |
+| Dropdown List Width<br/>`dropdownWidth`       | Text                                                               | "400px"         | Dropdown List Width Logic    | YES          | YES        | This handler allows you to specify the dropdown list width.            |
+| Dropdown List Position<br/>`dropdownPosition` | Select [Left:`left`<br/>Center:`center`<br/>Right:`right`]         | Center:`center` | Dropdown List Position Logic | YES          | YES        | This handler allows you to select the position of the dropdown.        |
+| Theme<br/>`theme`                             | Select [Light:`light`<br/>Dark:`dark`<br/>Auto:`auto`]             | Light:`light`   | Theme Logic                  | YES          | YES        | This handler allows you to select a theme for the emoji dropdown list. |
+| Emoji Style<br/>`emojiStyle`                  | Select [Apple:`apple`<br/>Google:`google`<br/>Facebook:`facebook`] | Apple:`apple`   | Emoji Style Logic            | YES          | YES        | This handler allows you to select an emoji style.                      |
+| Search Disabled<br/>`searchDisabled`          | Checkbox                                                           | `true`          | Search Disabled Logic        | YES          | YES        | This handler allows you to disable the emoji search field.             |
+
```

Comment:
Is it easy and obvious to implement binding emojis with default input? If no - need an instruction how to do it.